arangod/IResearch/IResearchMMFilesLink.h

```diff
@@ -60,10 +60,6 @@ class IResearchMMFilesLink final : public arangodb::MMFilesIndex, public IResear
   //////////////////////////////////////////////////////////////////////////////
   static arangodb::IndexTypeFactory const& factory();
 
-  virtual bool hasBatchInsert() const override {
```

Comment:
does MMFilesIndex no longer support batchInsert?

Reply:
no one ever calls this outside of catch tests